docs/developer_guide/SimpleSensorSimulator.md

```diff
@@ -2,25 +2,98 @@
 
 ![simple sensor simulator](../image/simple_sensor_simulator.png "simple sensor simulator")
 
-The simple sensor simulator is a reference implementation of the simulator which follows our scenario testing framework.
-This package includes very, very simple lidar simulation and send simulated detection result to the Autoware.
+The simple sensor simulator is a reference implementation of the simulator which follows our scenario testing framework.  
+This package includes very simple sensor or detection result simulation.
+
+Our simple sensor simulators do not include noise simulation, because scenario_simulator_v2 is a testing framework for planners or controllers instead of any perception modules. 
+
+
+[//]: # (This package includes very, very simple lidar simulation and send simulated detection result to the Autoware.)
 
 <font color="#065479E">_Note! Simple Sensor Simulator is just a reference implementation, so we can adapt any kinds of autonomous driving simulators if we can develop ZeroMQ interface to your simulator._</font>
 
+
+## LiDAR Simulation
+With this simulation, we can get lidar point-cloud data based on simple ray-casting algorithm.
+
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachLidarSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_lidar_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5563 |  
+
+### Configuration
```

Comment:
Please remove it.
Configuration should be auto-generated from protobuf-gen-c-doc

Reply:
OK. I'll replace it with the link to [auto-generated document](https://tier4.github.io/scenario_simulator_v2-docs/proto_doc/protobuf/) 
But, I think the unit information of each item in the configuration is the minimum required.
So, please check unit colomn!
After that, I'll add the unit information to `.proto` files. 
(This means unit information will be added to auto-generated documents)